incubator/java-openliberty/image/project/validate.sh

```diff
@@ -1,15 +1,25 @@
 #!/bin/bash
 
+#can we even run the appsody command?
+if [ -e /project/user-app/.appsody-nodev ]
+then 
+	if [[ ! -z $APPSODY_DEV_MODE && $APPSODY_DEV_MODE == 'prep' ]]
+	then
+		echo appsody run/debug/test not supported when .appsody-nodev detected.
+		exit 1
```

Comment:
Hmm... if a key point is to quietly make it through a pipeline including a `appsody test`... exiting with '1' might be bad.   Maybe we should try to get more input in this caes.